styles.css

```diff
@@ -11,3 +12,32 @@ img {
   bottom: 0;
   right: 0;
 }
+
+
+.direction1 {
+  transform: rotate(45deg);
+}
+
+.direction2 {
+  transform: rotate(90deg);
+}
+
+.direction3 {
+  transform: rotate(135deg);
+}
+
+.direction4 {
+  transform: rotate(180deg);
+}
+
+.direction5 {
+  transform: rotate(225deg);
+}
+
+.direction6 {
+  transform: rotate(270deg);
+}
+
+.direction7 {
+  transform: rotate(315deg);
+}
```

Comment:
If you want the car to move in 8 directions that is fine, however you only need four.

Numbers are harder to read, so feel free to use things like `.direction.top-right` and `direction.top-left`

Again, 4 directions would be much better for MVP.